README.md

```diff
@@ -22,13 +22,26 @@ If you'd like to remove the "Powered by Metabase" attribution, check out our [En
 
 We'll first need to set up a running instance of Metabase to serve the embedded dashboards.
 
-1. If you already have an instance of Metabase running on your machine, shut it down. Then download and run Metabase according to the guides in our [install section](https://www.metabase.com/docs/latest/operations-guide/installing-metabase.html).
+1. Shut down Metabase if you already have it running on your machine.
+2. Download the [JAR file for Metabase OSS](https://www.metabase.com/start/oss/jar).
+3. Open up a terminal and clone this repo to your machine.
+4. Move the `metabase.jar` file to the `metabase` directory of this repo.
+5. Run the JAR file:
```

Comment:
Might want to have them change into the metabase directory first before running the command.